website/sidebars.js

```diff
@@ -3,145 +3,145 @@
  - create an ordered group of docs
  - render a sidebar for each doc of that group
  - provide next/previous navigation
-
  The sidebars can be generated from the filesystem, or explicitly defined here.
-
  Create as many sidebars as you want.
  */
 
 module.exports = {
   // By default, Docusaurus generates a sidebar from the docs folder structure
   // conceptsSidebar: [{type: 'autogenerated', dirName: '.'}],
 
-    // But you can create a sidebar manually
-    sidebar: [
+  // But you can create a sidebar manually
+  sidebar: [
```

Comment:
we seriously need to add a formatting hook to commits as every person keeps formatting this file differently.